localization/english/bookmark/bookmark_508_l_english.yml

```
@@ -1,11 +1,11 @@
 ﻿l_english:
  bm_the_gnoll_war:0 "Gnoll War"
- bm_the_gnoll_war_desc:0 "PH"
+ bm_the_gnoll_war_desc:0 "After the collapse of Arathor annd the division of humanity into several kingdoms and city-states, settlers from Arathi migrated to the southern land of Azeroth and founded the city of Stormwind. Led by the noble Wrynn dynasty,the kingdom became a large and prosperous nation but the new king Barathen have now to face the rise of another ambitious ruler, Garfang, leader of the largest gnoll pack ever seen and determined to conquer and plunder the rich human lands. Will Stormwind maintain its place as the dominant power of the sub-continent or is it time for the mongrel gnolls to finally rule ? "
```

Just some typos here and there:
```suggestion
 bm_the_gnoll_war_desc:0 "After the collapse of Arathor and the division of humanity into several kingdoms and city-states, settlers from Arathi migrated to the southern land of Azeroth and founded the city of Stormwind. Led by the noble Wrynn dynasty, the kingdom became a large and prosperous nation. The new king Barathen now has to face the rise of another ambitious ruler, Garfang, leader of the largest gnoll pack ever seen and determined to conquer and plunder the rich human lands. Will Stormwind maintain its place as the dominant power of the sub-continent or is it time for the mongrel gnolls to finally rule? "
```